src/components/views/createplayer/LandingPage.scss

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
+$border-radius: 1.5em;
+
+@keyframes mover {
+  0% { transform: translateY(0) scaleY(-1); }
+  50% {transform: translateY(45em) scaleY(-1); }
+  51% {transform: translateY(45em) scaleY(1); }
+  100% { transform: translateY(0em) scaleY(1); }
+}
+
+body{
+  overflow: hidden;
+}
+
+.box {
+  display: flex;
+  margin: 50px;
+  margin-left: 400px;  //Set to center
+  height: calc(50vh);
+  align-items: center;
+  justify-content: center;
+}
+
+.box div {
+  width: 100px;
+  height: calc(40vh);
+  box-shadow: 0 0 0.5em black;
+}
+
+.box-right div {
+  width: 300px;
+}
+
+/* Create two equal columns that sits next to each other */
```

Comment:
Is this the image and the form?
equal in what?

Reply:
Yes